x-pack/legacy/plugins/oss_telemetry/server/lib/collectors/csp/csp_collector.test.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import sinon from 'sinon';
+import {
+  DEFAULT_CSP_RULES,
```

Comment:
Can you clarify? I used the defaults specifically to avoid that problem: when we update the defaults in the csp module, these tests will automatically be using the updated values.

Reply:
Ah wait, I see what you're saying...

Reply:
I can change the test defaults for the boolean flags, but we need to use `DEFAULT_CSP_RULES` for the rules-related tests because that's the value that the collector uses internally to determine `rulesChangedFromDefault`.